pkg/agent/statusupdate.go

```diff
@@ -53,8 +61,18 @@ func (su *periodicStatusUpdate) Execute() error {
 
 	// get the status from the health check and jobs
 	status := su.getCombinedStatus()
+	log.Debugf("Type of agent status update being checked %s : ", su.typeOfStatusUpdate)
```

Comment:
Have this here for debug purposes. I can be persuaded to remove this since the immediate status will execute every 10 secs.

Reply:
That or make it trace